app/services/import_service.py

```
@@ -647,35 +914,34 @@ def _import_media(
                 self.db.add(media)
                 if record_mapping and media_dto.external_id:
                     record_mapping("media", media_dto.external_id, media.id)
+
+                log_info(
+                    f"Media deduplicated during import",
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)